.rubocop.yml

```diff
@@ -0,0 +1 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
This project uses [standard](https://github.com/standardrb/standard), so I don't think we should have any rubocop config.

That might be a good thing to mention in the CONTRIBUTING.md (it's mentioned in the thoughtbot style guide that they use standard, but I don't think that's the first place somebody would look for that information).

Reply:
I'll remove them... I just noticed I had to do the ignores via standard...

Reply:
I removed the rubocop files and added the ignore via standard-todo